src/index.d.ts

```diff
@@ -361,8 +361,20 @@ declare namespace RamdaAdjunct {
         /**
          * Merge objects under corresponding paths.
          */
-        mergePaths(ps: Array<Array<string | number>>, obj: object): object;
-        mergePaths(ps: Array<Array<string | number>>): (obj: object) => object;
+        mergePaths(paths: Array<Array<string | number>>, obj: object): object;
+        mergePaths(paths: Array<Array<string | number>>): (obj: object) => object;
+
+        /**
```

Comment:
Create a new object with the own properties of the object under the `p` merged with the own properties of the provided `source`. If a key exists in both objects, the value from the `source` object will be used.